app/models/storage_location.rb

```diff
@@ -211,10 +217,18 @@ def self.csv_export_headers
   end
 
   def csv_export_attributes
-    [name, address, square_footage, warehouse_type, size]
+    attributes = [name, address, square_footage, warehouse_type, total_active_inventory_count]
+    get_active_items.sort { |a, b| a.item.name <=> b.item.name }.each { |item| attributes << item.quantity }
+    attributes
   end
 
   def empty_inventory_items?
     inventory_items.map(&:quantity).uniq.reject(&:zero?).empty?
   end
+
+  def get_active_items
```

Comment:
Ruby idiom is not to start a method with `get`. In this case you can just call it `active_items`.

Reply:
I ended up renaming it `active_inventory_items` to just be extra clear